config/locales/en/activemodel.yml

```
@@ -978,8 +978,9 @@ en:
               part_employ_evidence_missing: Upload the partner's employment evidence
               court_application_or_order_missing: Upload the application to court or the court order for Section 8 proceedings
               court_application_missing: Upload the application to court for Section 8 proceedings
-              court_order_missing: Upload the court order for Section 8 proceedings
+              court_order_missing: Upload the court order %{matter_type}
```

nit
```suggestion
              court_order_missing: Upload the court order%{matter_type}
```
and add space in the suffix/matter_type arg. Currently you would have a space at the end of court order